src/docs/development/frameworks-mobile/react-native.mdx

```diff
@@ -57,19 +68,33 @@ yarn add @pxblue/react-native-components
 If you are using the PX Blue component library, it includes basic support for theming your application. By default, it uses a Material-style theme. To use the PX Blue theme, first install it:
 
 ```
+npm install --save @pxblue/react-naive-themes
+or
 yarn add @pxblue/react-native-themes
 ```
 
-> Using the PX Blue React Native theme **requires** that you add the Open Sans font to your application. You can learn how to do this by reading the instructions for [Vanilla React Native](https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e) or [Expo](https://docs.expo.io/guides/using-custom-fonts/).
+> Using the PX Blue React Native theme **requires** that you add the Open Sans font to your application. You can learn how to do this by reading the instructions for [Vanilla React Native](https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e) or [Expo](https://docs.expo.io/guides/using-custom-fonts/). You can find the fonts [here](https://github.com/pxblue/cli-helpers/tree/master/fonts).
+
 > When using Expo, you will need to specify the name for each font weight you load using the format OpenSans-`<Weight>`, e.g., OpenSans-SemiBold. Refer to one of our React Native demos for reference.
 
 ## Applying the Theme
 
-You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object. For detailed information, check [our documentation](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--theme).
+You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object.
+
+```
```

Comment:
\```tsx

Reply:
I've added language type to all of the code blocks